provider/http/server_test.go

```diff
@@ -35,26 +38,15 @@ func TestManifest(t *testing.T) {
 
 	handler := new(pmock.Handler)
 	handler.On("HandleManifest", mock.Anything).Return(nil).Once()
+	client := new(cmock.Client)
 
 	withServer(t, func() {
-		err = Send(mani, signer, provider, deployment)
-		require.NoError(t, err)
-	}, handler)
-}
-
-func TestStatus(t *testing.T) {
-	handler := new(pmock.Handler)
-
-	withServer(t, func() {
-		resp, err := http.Get("http://localhost:3001/status")
```

Comment:
Let's continue to test this.  I know `status` is overloaded, maybe call it `healthcheck`.  It's used in `_run/multi`